qemu/tests/slof_boot.py

```diff
@@ -81,6 +82,10 @@ def _verify_boot_status(boot_dev, content):
             test.fail(fail_info)
         test.log.info(ret_info)
 
+    o = process.getoutput(params.get("check_slof_version")).strip()
+    error_context.context("Check the version of SLOF: '%s'" % o,
```

Comment:
It should be `test.log.info`, it does not relate to any context.

Reply:
Updated it and could you help to review it again ? Thanks a lot. @PaulYuuu